OsmAnd/src/net/osmand/plus/auto/SurfaceRenderer.java

```diff
@@ -371,7 +371,7 @@ public void setupOffscreenRenderer() {
 							offscreenMapRendererView.addListener(this);
 							mapView.getAnimatedDraggingThread().toggleAnimations();
 						}
-						mapView.applyBatterySavingModeSetting(offscreenMapRendererView);
```

Comment:
disable happens in another view? or not needed?

Reply:
it reverts back to app setting when android auto goes off and switches back to device screen